ocean_provider/utils/accounts.py

```diff
@@ -42,6 +43,16 @@ def verify_signature(signer_address, signature, original_msg, nonce):
         new_signature = signature_bytes
 
     signature = keys.Signature(signature_bytes=new_signature)
+
+    if old_signature != str(signature):
+        msg = (
+            f"Invalid signature. Please check the nonce or documentId from the original message."
+            f" In case of compute endpoints, check also the job ID."
+            f" Got: {old_signature}\n. Expected: {signature}"
```

Comment:
not sure we should expose the correct signature... @alexcos20 what do you think?

Reply:
@mariacarmina since this is the only blocker for this PR, can you please remove "Expected: {signature}" and create a new issue to readd it if necessary?

Reply:
created https://github.com/oceanprotocol/provider/issues/616.